plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -510,6 +516,11 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    bool                                              _disable_subjective_p2p_billing              = true;
    bool                                              _disable_subjective_api_billing              = true;
    fc::time_point                                    _irreversible_block_time;
+   fc::time_point                                    _accepted_block_time;
```

Comment:
Can we rename this `_last_accepted_block_time`?

Reply:
This matches `_irreversible_block_time` in naming and I would prefer not to rename `_irreversible_block_time`.